### ISSUE TITLE
Fix heading levels

### ISSUE THREAD

**Issue by cncoulter** [cncoulter/Graph_Paper#8]
Use only one h1 element per page. Heading elements should be in a logical sequence. Don't skip heading levels. [WCAG 2.4.6](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-descriptive.html) is relevant to this.

There are issues on the following pages:

* https://graphpaper.cncoulter.com/about/
* https://graphpaper.cncoulter.com/people/
* https://graphpaper.cncoulter.com/people/Cam-Coulter/
* https://graphpaper.cncoulter.com/blog/
* https://graphpaper.cncoulter.com/2020/05/30/Welcome-to-Graph-Paper/
* https://graphpaper.cncoulter.com/blog/categories/
* https://graphpaper.cncoulter.com/blog/tags/
* https://graphpaper.cncoulter.com/fork/
* https://graphpaper.cncoulter.com/contribute/

Many of these issues can be fixed by using class="h5", or something like that.

**Comment by cncoulter** [comment]
Fixed!

https://github.com/cncoulter/Graph_Paper/commit/f5ebc4dff01c6c41f773acc27e89169fcd79ea3f
